src/docs-website/src/css/custom.css

```
@@ -330,11 +338,25 @@ pre code {
   /* Match p element typography */
   font-size: 0.875rem;
   font-weight: 400;
   line-height: calc(1.25 / 0.875);
+
+  /* .menu__link is itself a flex container (label text + caret pseudo-element).
+     Flex items default to min-width: auto, which stops long labels from
+     wrapping in some browsers (notably Safari) and lets them overflow past
+     the sidebar's clipped edge instead - taking the caret icon with them. */
+  min-width: 0;
+  white-space: normal;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)